src/modules/transaction/components/TxSignatureCollector/TxSignatureCollector.js

```diff
@@ -1,95 +1,148 @@
 import React, { useEffect } from 'react';
-import { signatureCollectionStatus } from '@transaction/configuration/txStatus';
+import { useDispatch } from 'react-redux';
+// import { signatureCollectionStatus } from '@transaction/configuration/txStatus';
 import { TertiaryButton } from 'src/theme/buttons';
+import { secondPassphraseRemoved } from '@auth/store/action';
+import { useCommandSchema } from '@network/hooks';
 import Icon from 'src/theme/Icon';
 import Box from 'src/theme/box';
 import Illustration from 'src/modules/common/components/illustration';
 import BoxContent from 'src/theme/box/content';
 import { isEmpty } from 'src/utils/helpers';
 import EnterPasswordForm from 'src/modules/auth/components/EnterPasswordForm';
+import { useAuth } from '@auth/hooks/queries';
 import { getDeviceType } from '@wallet/utils/hwManager';
 import { useCurrentAccount } from '@account/hooks';
 import styles from './txSignatureCollector.css';
+import { joinModuleAndCommand } from '../../utils';
+import { MODULE_COMMANDS_NAME_MAP } from '../../configuration/moduleCommand';
+import useTxInitatorAccount from '../../hooks/useTxInitiatorAccount';
 
+// eslint-disable-next-line max-statements
 const TxSignatureCollector = ({
   t,
   transactions,
-  account,
   actionFunction,
   multisigTransactionSigned,
-  rawTx,
+  formProps,
+  transactionJSON,
   nextStep,
   prevStep,
   statusInfo,
-  sender,
-  signatureStatus,
-  signatureSkipped,
+  transactionDoubleSigned,
   fees,
   selectedPriority,
 }) => {
-  const deviceType = getDeviceType(account.hwInfo?.deviceModel);
-  const [currentAccount] = useCurrentAccount();
+  const [sender] = useCurrentAccount();
+  const { moduleCommandSchemas } = useCommandSchema();
 
+  // here, we want to get the auth account details of the user presently wanting to sign the transaction
+  const { data: account, isLoading: isGettingAuthData } = useAuth({
+    config: { params: { address: sender.metadata.address } },
+  });
+
+  // here, we want to get the auth account details of the account that initated the transaction.
+  const { isLoading: isGettingTxInitatorAccount, txInitatorAccount } = useTxInitatorAccount({
```

Comment:
@eniolam1000752 Please fix typo in `isGettingTxInitatorAccount` and `txInitatorAccount` and related uses